RedditSharp/Reddit.cs

```diff
@@ -265,7 +265,7 @@ public AuthenticatedUser GetMe()
         /// <returns></returns>
         public Subreddit GetSubreddit(string name)
         {
-            name = System.Text.RegularExpressions.Regex.Replace(name, "(r/|/)", "");
+            name = System.Text.RegularExpressions.Regex.Replace(name, "^([/](r/))|/", "");
```

Comment:
"r/subredditr/" => "rsubredditr"

Hence why I had the optional tag ( `?` ) after the `[/]` in my regex